### DISCUSSION TITLE
Can I switch all open windows from the laptop to the connected screen at once without focusing each individually?

### DISCUSSION THREAD

**Discussion by Matlor** [rxhanson/Rectangle#1237] [Ideas]
So I use the "next display" feature often but have to use the command on each window individually. It would be really nice if I could do this for all windows at once. Could this be possible?

**Comment by rxhanson** [comment]
Thanks or the feedback. I don't have plans to add something like this into Rectangle, but would be fine merging in a pull request that provides this functionality (as long as it is not added to the UI). 

[Rectangle Pro](https://rectangleapp.com/pro) (paid) does have an action for moving all windows belonging to an app to the next/previous display, although that's not precisely what you're looking for.
